x-pack/plugins/saved_objects_tagging/public/management/components/_action_bar.scss

```diff
@@ -0,0 +1,19 @@
+.tagMgt__actionBar {
+  > .euiHorizontalRule {
+    margin-bottom: 0;
+  }
+
+  .euiButtonEmpty {
+    height: 18px;
+    font-size: 0.75rem;
+
+    .euiIcon {
+      height: 12px;
+      width: 12px;
+    }
+  }
```

Comment:
Let's not try to force **create** a new sized button on top of EuiButtonEmpty here. Instead, you can simply use a link inside of an XS text block like:

```
<EuiText size="xs">
  <EuiLink onClick={}><EuiIcon size="s" /> Text </EuiLink>
</EuiText>
```

Reply:
Well, in term of semantic and accessibility, this is a button, not a link.

Reply:
EuiLink when provided an onClick will render a <button> element, so symantically the rendered HTML will be correct.

Reply:
https://github.com/elastic/kibana/pull/82816/commits/1452ccc6a361194283df066f5dc506d1e0ff4a86